hsclient/hydroshare.py

```diff
@@ -562,14 +608,17 @@ def file_aggregate(self, path, agg_type: AggregationType):
 
         url = urljoin(self._hsapi_path, "functions", "set-file-type", path, type_value)
         self._hs_session.post(url, status_code=201, data=data)
-        self.refresh()
-        return self.aggregation(file__path=path)
+        if refresh:
+            self.refresh()
+            return self.aggregation(file__path=path)
```

Comment:
Was this indention intentional? Either `return self.aggregation(file__path=path)` or `None`? Just wanted to double check.

Reply:
With refresh toggled off, we lack the metadata to return the aggregation, so yes this is intentional.  I'll add a comment.